src/Simplex/Messaging/Agent/Protocol.hs

```diff
@@ -967,7 +982,6 @@ data AMsgType
 instance Encoding AMsgType where
   smpEncode = \case
     HELLO_ -> "H"
-    REPLY_ -> "R"
```

Comment:
keep commented for some time to not use in near future?

Reply:
though we usually check ability to use new features via version, meaning if it's sent other side should also have it deleted.. so probably it's ok